src/levellistloader_arm9.asm

```diff
@@ -18,9 +18,9 @@
 ;First do am9.bin
 ;.open "../bin_src/arm9.bin", "../bin_out/arm9.bin", 0x02000000
     ;Implement our own file loading function to load the list as a sir0!
-    .org 0x020A46EC ;write over the actual table
+    .org LvlListTable ;write over the actual table
     ;.area 0x20A6910 - .;0x2224 ;We got at most 8,740 bytes to write stuff here, so there's plenty of room!
-    .area 0x20A68BC - .
+    .area 0x21D0, 255 ; We null out the rest of the table, so we know something went wrong
```

Comment:
Yeah, but the . operator is the current offset. So, it will adapt to whatever pos its currently starting at. The code before wasn't right, because it used exclusively the NA offset. But it might be better to keep the subtraction there.